[ 2025-06-12 13:24:30,409 ] root - INFO - Entered the start_data_ingestion method of TrainPipeline class
[ 2025-06-12 13:24:30,409 ] root - INFO - Getting the data from mongodb
[ 2025-06-12 13:24:30,409 ] root - INFO - Entered initiate_data_ingestion method of Data_Ingestion class
[ 2025-06-12 13:24:30,409 ] root - INFO - Exporting data from mongodb
[ 2025-06-12 13:24:30,409 ] root - ERROR - Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/configuration/mongo_db_connection.py] at line number [51]: Environment variable '' is not set.
[ 2025-06-12 13:24:30,409 ] root - ERROR - Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/data_access/vehicle_data.py] at line number [20]: Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/configuration/mongo_db_connection.py] at line number [51]: Environment variable '' is not set.
[ 2025-06-12 13:24:30,410 ] root - ERROR - Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/components/data_ingestion.py] at line number [34]: Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/data_access/vehicle_data.py] at line number [20]: Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/configuration/mongo_db_connection.py] at line number [51]: Environment variable '' is not set.
[ 2025-06-12 13:24:30,410 ] root - ERROR - Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/components/data_ingestion.py] at line number [86]: Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/components/data_ingestion.py] at line number [34]: Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/data_access/vehicle_data.py] at line number [20]: Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/configuration/mongo_db_connection.py] at line number [51]: Environment variable '' is not set.
[ 2025-06-12 13:24:30,410 ] root - ERROR - Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/pipeline/training_pipeline.py] at line number [47]: Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/components/data_ingestion.py] at line number [86]: Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/components/data_ingestion.py] at line number [34]: Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/data_access/vehicle_data.py] at line number [20]: Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/configuration/mongo_db_connection.py] at line number [51]: Environment variable '' is not set.
[ 2025-06-12 13:24:30,410 ] root - ERROR - Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/pipeline/training_pipeline.py] at line number [133]: Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/pipeline/training_pipeline.py] at line number [47]: Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/components/data_ingestion.py] at line number [86]: Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/components/data_ingestion.py] at line number [34]: Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/data_access/vehicle_data.py] at line number [20]: Error occurred in python script: [/home/vishalr/Desktop/Vehicle_insurance_prediction/src/configuration/mongo_db_connection.py] at line number [51]: Environment variable '' is not set.
